packaging/nnstreamer.spec

```diff
@@ -5,6 +5,8 @@
 %define		tensorflow_support	0
 %define		armnn_support 0
 %define		python_support 1
+%define		pytorch_support 0
+%define		caffe2_support 0
```

Comment:
how about adding 'tensorflow_lite_support'?

Reply:
I thought supporting tf-lite in Tizen is default. But adding tensorflow_lite_support is a good idea.

Reply:
@jaeyun-jung 'tensorflow_lite_support' is added!